tests/php/Security/MemberAuthenticatorTest.php

```diff
@@ -109,36 +57,46 @@ public function testCustomIdentifierField()
 
     public function testGenerateLoginForm()
     {
+        $authenticator = new Authenticator();
+
         $controller = new Security();
 
         // Create basic login form
-        $frontendForm = MemberAuthenticator::get_login_form($controller);
-        $this->assertTrue($frontendForm instanceof MemberLoginForm);
+        $frontendResponse = $authenticator
+            ->getLoginHandler($controller->link())
+            ->handleRequest(new HTTPRequest('get', '/'), \SilverStripe\ORM\DataModel::inst());
+
+        $this->assertTrue(is_array($frontendResponse));
```

Comment:
Can we use proper assertions, please (`assertInternalType`, `assertArrayHasKey`, `assertInstanceOf`)